docs/source/_static/custom.js

```diff
@@ -29,6 +29,7 @@ document.addEventListener('DOMContentLoaded', () => {
         { selector: '.toctree-l1 > a', text: 'Managed Jobs' },
         { selector: '.toctree-l1 > a', text: 'Running on Kubernetes' },
```

Comment:
maybe can remove this?

Reply:
Let's keep Kubernetes, as we want to promote k8s use cases?